### PR TITLE
Support djagno 1.8

### PR DESCRIPTION
**Pull request by jlward**

### CODE REVIEW

**Comment by jlward** [comment]
I would hold off on reviewing this until #1 gets merged in.

File: tests/test_templates.py [hunk 1]
```diff
@@ -7,10 +7,16 @@
...
 """
 
-from django.template import Template, Context, add_to_builtins
+from django.template import Template, Context
 from django.http import HttpRequest
 import django_tables as tables
 
+try:
+    from django.template import add_to_builtins
+except ImportError:
```
Comment (winhamwr) [inline tests/test_templates.py]: Let's add a code comment here explaining that django 1.X moved this.

File: django_tables/base.py [hunk 2]
```diff
@@ -2,12 +2,23 @@
...
+from django.utils.encoding import force_unicode
 from django.utils.text import capfirst
 from columns import Column
 from options import options
 
 
+try:
+    from django.utils.encoding import StrAndUnicode
+except ImportError:
+    from django.utils.encoding import python_2_unicode_compatible
```
Comment (winhamwr) [inline django_tables/base.py:14]: Needs code comment explaining that Django 1.X removed this.

**Comment by winhamwr** [comment]
:+1:

**Pull request merged**